src/index.js

```diff
@@ -62,7 +62,15 @@ const init = async () => {
                 payload: Joi.object({
                     for: Joi.string().required().valid('cdb'),
                     new: Joi.boolean().required(),
-                    data: Joi.object().required(),
+                    data: Joi.object({
+                        slug: Joi.string().required(),
+                        'relevant-countries': Joi.array().items(Joi.string()).required(),
+                        name: Joi.string().required(),
+                        sources: Joi.array().items(Joi.string()).required(),
+                        address: Joi.string().required(),
+                    })
+                        .required()
```

Comment:
Actually, we can't do that. The suggest explicitly doesn't require any fields. As long as users fill in one single field, we treat that as a valid suggestion (and this does have at least one useful application: 'suggesting' companies as in 'please add this company').

Reply:
That’s not true. Actually, the client already does a lot of post-processing to fill in non-filled out fields. We need a slug at the very least, and for that we need either a name or a website. Anything else is just stupid, cause what are we supposed to do with just an address? I agree we won‘t need to require the address or the sources (though I would still like to validate that, as it breaks the code). If you want, I could try to present human-readable, explainatory error messages.

Reply:
Yes, it is. :P We _do_ accept suggestions with just one field filled in but as you correctly point out, we automatically add the slug and relevant countries if necessary.

I also agree that we *should* probably require a name. That is not currently implemented in website, though.

Can't you just define defaults in Joi instead of requiring the fields? That should be enough to not make the code break.